dsperse/src/cli/slice.py

```
@@ -33,8 +35,85 @@ def setup_parser(subparsers):
+                break
+            print(f"{Fore.YELLOW}Invalid choice: '{chosen}'. Please enter one of: dirs, dslice, dsperse{Style.RESET_ALL}")
+
+    try:
+        # Special handling: dsperse -> dirs with optional expand toggle
+        if p.is_file() and p.suffix == '.dsperse' and getattr(args, 'to_type', None) == 'dirs':
+            result = Converter._dsperse_to_dirs(p, output_path, expand_slices=bool(getattr(args, 'expand_slices', False)))
+            if bool(getattr(args, 'cleanup', True)):
+                try:
+                    p.unlink()
+                except Exception:
+                    pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)